omod/pom.xml

```diff
@@ -209,6 +209,13 @@
 			<groupId>org.openmrs.module</groupId>
 			<artifactId>uicommons-api</artifactId>
 		</dependency>
+
+		<dependency>
+			<groupId>org.openmrs.module</groupId>
+			<artifactId>uicommons-omod</artifactId>
+			<version>${uicommonsVersion}</version>
+			<scope>provided</scope>
+		</dependency>
```

Comment:
I don't think this is necessary, can you double check?

Reply:
Indeed it's not needed. Removed changes from pom.xml